docs/moderator.md

```diff
@@ -0,0 +1,86 @@
+> Hello!
+>
+> Welcome to the Angular Community team! We're glad you're here. You're helping
+> ensure the community stays welcoming for everyone. Below, you will find an
+> overview of the server and a frequently asked questions of the most common
+> scenarios you will run into.
+>
+> If you have any questions about the server or need help with anything, please
+> do not hesitate to reach out to myself or to another `@Staff` member.
+>
+> Sincerely,
+>
+> delasteve
+
+## Preface
+
+Before we begin, you'll need to enable the `Developer Mode` feature in Discord. You
+can find how to do that [here][discord-developer-mode]. This will help ensure
+you're able to view user ids in the event you have to take action against a member
+who violates our [Code of Conduct (CoC)](../CODE_OF_CONDUCT.md) or
+[server rules](../README.md). To find the id of a user, simply right click their
+username and select `Copy ID` from the menu that appears. This also works for just
+about everything, such as messages, channels, and servers.
+
+## Server Overview
+
+Upon getting the `@Moderator` role, the first thing you'll notice is many new
+channels. These are set up for logging purposes. Hopefully, we will never need
+to use them, but they prove invaluable when we do. It's recommended to either mute
+the entire category or individual channels, depending on your preference, then
+collapse the category.
+
+You will also see a `Private` category. This is where we chat amongst the other team
+members. Some team members have notifications on for all messages sent to a channel,
+specifically `#moderator-chat`. You're not required to do the same, but please be
+mindful of what channel you're sending messages in.
+
+Our moderation bot, `@Gaius Cicereius+`, takes care of most of the heavy lifting. It
+has the ability to filter unwanted words, check user profiles, reaction roles, and
+more. The documentation for the bot can be found [here][gaius-documentation].
+
+Our other bot, `@Statbot`, is a simple analytics bot. Its main goal is to keep track
+of the activity in certain channels. This will become helpful if we ever reach a
+point where we have too many channels and need to prune unused or unmaintained
+library channels. Our analytics are public and can be found
+[here][statbot-analytics].
+
+All bot commands should be used in the `#bot-commands` channel. This way we can
+keep the spam from the bots isolated from our actual private moderation channels.
+
+## FAQ
+
+> A user wants to opt-out of analytic collection from `@Statbot`. What do I tell them
+> to do?
+
+Direct them to send a DM the Statbot with the command `s?userprivcy on` and then
+type `accept` to the "Are you sure" message that follows.
+
+> The word was caught in the filter that should be allowed. How do I fix it so that
+> the word won't be caught in the future?
```

Comment:
`A word` was caught.

How do I fix this so that it won't be caught in the future?